### ISSUE TITLE
Reduce IOS video size

### ISSUE THREAD

**Issue by DorianMazur** [mobile-dev-inc/maestro#1933]
### Use case

Video file sizes on iOS are nearly ten times larger than on Android, leading to increased storage needed (Github Actions Artifacts)

### Proposal

Optimize video encoding settings on iOS by adjusting the bitrate, resolution, and possibly switching to more efficient codecs like H.265. Additionally, consider implementing background compression post-recording to further reduce file sizes.

**Comment by bartekpacia** [comment]
Hey Dorian,

What command do you have in mind? `maestro record`, or recording provided by Maestro Cloud, or both?

About the problem: I think we could save a lot of storage by converting `.mov` to `.mp4`, or even better, `.av1`.

**Comment by DorianMazur** [comment]
Yes, I think we should add a config option to make the video files smaller for IOS. Converting the videos from .mov to .mp4 using ffmpeg should help. The difference is big—on Android, a 2-minute video is about 4MB, while on iOS, the same video is around 45MB.

**Comment by bartekpacia** [comment]
Yeah, this makes sense. What command do you have in mind? `maestro record`, or the recording provided by Maestro Cloud, or both?

**Comment by bartekpacia** [comment]
This is a valid issue.

**Comment by DorianMazur** [comment]
I can create PR to implement this. Currently we are using ffmpeg to reduce video size by 90% before upload as GitHub artifacts.

**Comment by Fishbowler** [comment]
Which video files are we talking about here?
The ones that come from `- startRecording` ?

**Comment by sergionettl** [comment]
Agree, there is no reason for the bitrate to be so high for the recordings.
Adding an option to set the bitrate for the command `startRecording` would be helpful

**Comment by Fishbowler** [comment]
Do we need config? Or does someone just need to open a PR to set the bitrate lower?

**Comment by nanihadesuka** [comment]
Both are fine, whatever is easier
